src/replication.c

```diff
@@ -205,25 +205,27 @@ void feedReplicationBufferWithObject(robj *o) {
     feedReplicationBuffer(p, len);
 }
 
-/* Generally, we only one replication buffer block when replication backlog
- * size exceeds our setting and no slave reference it. But if slaves clients
- * disconnect, we need to free the replication buffer blocks that are referenced,
- * but it would cost much time if there are a lots blocks to free, that will
+/* Generally, we only have one replication buffer block to trim when replication backlog
+ * size exceeds our setting and no replica reference it. But if replica clients
+ * disconnect, we need to free many replication buffer blocks that are referenced.
+ * It would cost much time if there are a lots blocks to free, that will
  * freeze server, so we trim replication backlog incrementally. */
 void incrementalTrimReplicationBacklog(size_t max_blocks) {
     serverAssert(server.repl_backlog != NULL);
 
     size_t trimmed_blocks = 0, trimmed_bytes = 0;
     while (server.repl_backlog_histlen > server.repl_backlog_size &&
-           trimmed_blocks < max_blocks) {
-        /* We never trim backlog if only one block. */
+           trimmed_blocks < max_blocks)
+       {
```

Comment:
sorry, my bad (hard to control indentation in suggestions)
```suggestion
    {
```

Reply:
I am making code fine, and apply your suggestions, l will update